filer/tests/admin.py

```diff
@@ -2508,6 +2510,12 @@ def test_multi_files_perms_for_restricted_descendants(self):
         self.assertEqual(File.objects.filter(restricted=False).count(), 0)
         self.assertEqual(Folder.objects.filter(restricted=False).count(), 0)
 
+    def test_truncate_filename(self):
```

Comment:
We should also that if the user supplied an extension, that is honored.

Reply:
sure, can do